config/default.yml

```diff
@@ -213,6 +213,13 @@ Rails/HasManyOrHasOneDependent:
   Include:
     - app/models/**/*.rb
 
+Rails/HashEnum:
+  Description: 'Prefer hash syntax over array syntax when defining enums'
```

Comment:
Please add a period at the end of the sentence and add `StyleGuide: '#enums'`.

```diff
- Description: 'Prefer hash syntax over array syntax when defining enums'
+ Description: 'Prefer hash syntax over array syntax when defining enums'.
+ StyleGuide: '#enums'
```

https://rails.rubystyle.guide/#enums